src/Tests/Consolonia.Gallery.Tests/TransitioningContentTests.cs

```diff
@@ -18,7 +18,7 @@ public async Task PerformSingleTest()
             await UITest.KeyInput(Key.Tab);
             await UITest.KeyInput(Key.Tab);
             await UITest.KeyInput(Key.Space);
-            await Task.Delay(500); // Wait for the animation to finish
+            await Task.Delay(1000); // Wait for the animation to finish
```

Comment:
Should be 500 based on the gallery animatin. Lets open a constant in -> `private int _duration = 500;` and re-use it here.

If it's not enough, I guess we can just add await WaitRendered();

Reply:
yeah sometimes 500 was ok, sometimes it's slow. I would love a more canonical way of knowing when to print the buffer.